src/monitoring/logflare.ts

```diff
@@ -20,5 +20,8 @@ export default function () {
   return createLogFlareWriteStream({
     apiKey: logflareApiKey,
     sourceToken: logflareSourceToken,
+    onPreparePayload: (payload: any) => {
+      return payload
```

Comment:
<img width="1067" alt="Screenshot 2023-09-29 at 8 25 10 PM" src="https://github.com/supabase/storage-api/assets/22714384/5c61c9a6-3809-4231-b7dc-07089a36fff6">

would need to set the project key here 

https://github.com/Logflare/pino-logflare/blob/master/docs/API.md

Reply:
@Ziinc the project key is already inferred from the log-middleware https://github.com/supabase/storage-api/pull/371/files#diff-7657d30e7d33d62772ae092233fbe7844f1601fbda2d5ebe6252b220988928abR7

do you think is still worth calling `defaultPreparePayload(payload, meta)`?

If yes, would this make sense:

```ts
onPreparePayload: (payload: any, meta: any) => {
      const item = defaultPreparePayload(payload, meta)
      item.project = payload.project
      return item
    },
```

Reply:
Yea it is needed as the key needs to be a top level field for enabling clustering